test/benchmark/lmbench-udp-latency-localhost/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+echo "*** Running lmbench udp latency test ***"
+
+/benchmark/bin/lmbench/lat_udp -s
+/benchmark/bin/lmbench/lat_udp -P 1 127.0.0.1
```

Comment:
There is a shutdown command, right?

```
/benchmark/bin/lmbench/lat_udp -S
```

I think it would be better to shut down the UDP server properly.